connector.py

```diff
@@ -355,7 +360,17 @@ def _process_response(self, response):
         for r in response:
             if not cols:
                 cols = [(f, r._fields[f].db_type) for f in r._fields]
-            data.append([getattr(r, f) for f in r._fields])
+            vals = []
+            for fi in r._fields:
+                val = getattr(r, fi)
+
+                clickhouseUseLocalTimezone = int(os.getenv('CLICKHOUSE_USE_LOCAL_TIMEZONE', '0'))
```

Comment:
Can you set the variable in the module header with other settings (instead of inside the loop)?